workspace.code-workspace

```diff
@@ -102,7 +102,8 @@
       "preprocessors",
       "serializable",
       "surrialize"
-    ]
+    ],
+    "compile-hero.disable-compile-files-on-did-save-code": false
```

Comment:
What is this?

Reply:
That went in by mistake. It's a markdown plugin I have.